src/Components/Boolist/Book.js

```diff
@@ -0,0 +1,42 @@
+import '../Style/Style.css';
+import NewBookForm from './NewBookForm';
+
+const Book = () => {
+  const data = [
```

Comment:
- [OPTIONAL] Kindly make sure to name your component a well-detailed name like `BookList` instead the `Book` because I believe you are going to use the `Book` name to create a new component of the single book.